.github/helpers/contracts/register-ens-subdomain.js

```diff
@@ -0,0 +1,283 @@
+const ethers = require('ethers');
+
+async function main() {
+  if (process.argv.length < 5) {
+    console.log('register-ens-subdomain.ts DOMAINNAME SUBDOMAIN RESOLVER');
+    return;
+  }
+
+  const ENS_REGISTRY = process.env.ENS_REGISTRY;
+  const provider = new ethers.providers.InfuraProvider(
+    process.env.NETWORK,
+    '579ebd226d934b2fac48c7ffb0c1907c'
```

Comment:
Shouldn't this be in an env var?

Reply:
Also, I think package.json now has an env var where the domain is still `dao-plugin.eth`